Lib/test/test_inspect/test_inspect.py

```diff
@@ -811,6 +811,13 @@ def test_getfile_builtin_module(self):
             inspect.getfile(sys)
         self.assertStartsWith(str(e.exception), '<module')
 
+    def test_getfile_custom_module(self):
+        import types
```

Comment:
```suggestion
        import re
```

`types` module is imported at the top level. But `re` is needed only here.